### PR TITLE
Master dev kit

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,45 +1,68 @@
...
-    - SYMFONY_PHPUNIT_VERSION=5.6
+    - SYMFONY_DEPRECATIONS_HELPER=48
+    - SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_VERSION=5.7
+    - TEST_INSTALLATION=false
 
 matrix:
   include:
     - php: 7.1
-      env: SYMFONY_VERSION=3.3.* DEPS=dev
+      env: SYMFONY_VERSION=.*
```
Comment (ElectricMaxxx) [inline .travis.yml]: ooops, i need to have an other condition for compoenent here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 Contributing
 ------------
 
-Symfony CMF is an open source, community-driven project. We follow the same
-guidelines as core Symfony. If you'd like to contribute, please read the
+Symfony2 CMF is an open source, community-driven project. We follow the same
```
Comment (dbu) [inline CONTRIBUTING.md]: this should say "Symfony CMF", not Symfony2 CMF. the official name of symfony moved from Symfony2 to Symfony, as symfony 3 and 4 are still symfony.

File: README.md [hunk 3]
```diff
@@ -17,38 +28,55 @@ For the best integration into the Symfony full stack framework, it is
...
+For general support and questions, please use [StackOverflow](http://stackoverflow.com/questions/tagged/symfony-cmf).
 
 ## Contributing
 
 Pull requests are welcome. Please see our
 [CONTRIBUTING](https://github.com/symfony-cmf/symfony-cmf/blob/master/CONTRIBUTING.md)
 guide.
 
-Unit and/or functional tests exist for this component. See the
+Unit and/or functional tests exist for this bundle. See the
```
Comment (dbu) [inline README.md]: this is not a bundle, but a component. lets change the template to say `package` - that is correct for anything, be it a bundle or not.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ElectricMaxxx) [reply CONTRIBUTING.md]: This should be updated in mostly all packages then.

**Comment by dbu** [comment]
something is wrong with the test installation thingy (setup, not the test installation itself). the make target is not found...

**Comment by ElectricMaxxx** [comment]
ahh yes, i know why
having the installation test hard in allowed failures is not good, it should depend on the configuration. So if it is configured, cause the make target will be available then only.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,7 +2,7 @@ Contributing
 ------------
 
 Symfony CMF is an open source, community-driven project. We follow the same
-guidelines as core Symfony. If you'd like to contribute, please read the
+guidelines as core Symfony2. If you'd like to contribute, please read the
```
Comment (dbu) [inline CONTRIBUTING.md]: here it says Symfony2 as well

**Comment by dbu** [comment]
right, there is no point in testing symfony installation of a non-bundle :-)

**Pull request merged**